_data/packages.yaml

```
@@ -221,6 +221,18 @@
   repo_description: Crystal manifest in WinGet packages repository
   repology: winget
 
+- type: community
+  os: Windows
+  arch: [x86_64]
+  title: Msys2
```

The official name is in fullcaps, ditto for below

```suggestion
  title: MSYS2
```